frog/imports/client/Preview/index.js

```diff
@@ -155,8 +154,4 @@ class PreviewPage extends React.Component<any, any> {
 
 PreviewPage.displayName = 'PreviewPage';
 
-export default (props: any) => (
-  <ErrorWrapper>
-    <PreviewPage {...props} />
-  </ErrorWrapper>
-);
```

Comment:
Why did you remove the `ErrorWrapper`? Does this have something to do with the windowing component?

Reply:
I don't remember removing this. Maybe this was a change merged after I started this branch.